Public/css/style.css

```diff
@@ -449,6 +449,15 @@ section h2 {
     color: #616161;
 }
 
+/* no-content */
+
+.no-content-container {
```

Comment:
Could you please copy .form-container's styles to here? We used .form-container in the empty search text, it is better two of them become the same. You can check search.leaf file as a reference